src/tools/genpolicy/src/policy.rs

```diff
@@ -1028,3 +1030,20 @@ pub fn get_kata_namespaces(
 
     namespaces
 }
+
+fn encode_init_data(init_data: &kata_types::initdata::InitData) -> String {
+    // from https://github.com/microsoft/kata-containers/blob/7cd4e3278a21e2b2499e1380f5c4d81fa6ebf8d6/src/runtime/pkg/oci/utils.go#L939C1-L984C2
```

Comment:
I would just keep these two lines of code - without the verbose comments around these lines.

    let toml_str = toml::to_string(&init_data).unwrap();
    kata_types::initdata::create_encoded_input(&toml_str)

Reply:
updated